### PR TITLE
Feature/dagplej 77 taxonomy permissions

### PR DESCRIPTION
**Pull request by martinyde**
Changed permissions

### CODE REVIEW

File: config/sync/user.role.anonymous.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ dependencies: {  }
 _core:
   default_config_hash: j5zLMOdJBqC0bMvSdth5UebkprJB8g_2FXHqhfpJzow
 id: anonymous
-label: 'Anonymous user'
+label: 'Anonym bruger'
```
Comment (rimi-itk) [inline config/sync/user.role.anonymous.yml:8]: Do we really want this to be in Danish?

File: config/sync/user.role.authenticated.yml [hunk 2]
```diff
@@ -5,7 +5,7 @@ dependencies: {  }
 _core:
   default_config_hash: dJ0L2DNSj5q6XVZAGsuVDpJTh5UeYkIPwKrUOOpr8YI
 id: authenticated
-label: 'Authenticated user'
+label: 'Godkendt bruger'
```
Comment (rimi-itk) [inline config/sync/user.role.authenticated.yml:8]: Do we really want this to be in Danish?

File: composer.json [hunk 3]
```diff
@@ -86,7 +86,15 @@
...
-        "sort-packages": true
+        "sort-packages": true,
+        "allow-plugins": {
+            "dealerdirect/phpcodesniffer-composer-installer": true,
+            "composer/installers": true,
+            "cweagans/composer-patches": true,
+            "drupal/console-extend-plugin": true,
+            "drupal/core-composer-scaffold": true,
+            "drupal/core-project-message": true
+        }
```
Comment (rimi-itk) [inline composer.json:97]: Does this require an update of `composer.lock` (e.g. by running `composer update --lock`)?

**Review by rimi-itk — CHANGES_REQUESTED** [review]
Changing the permissions on the file `scripts/git/hooks/pre-commit` does not seem right.

File: composer.json (hunk 3 above)
Comment (martinyde) [reply composer.json:97]: Updated composer lock

File: config/sync/user.role.authenticated.yml (hunk 2 above)
Comment (martinyde) [reply config/sync/user.role.authenticated.yml]: Fixed

File: config/sync/user.role.anonymous.yml (hunk 1 above)
Comment (martinyde) [reply config/sync/user.role.anonymous.yml]: Fixed

**Review by rimi-itk — CHANGES_REQUESTED** [review]
The permissions on `scripts/git/hooks/pre-commit` should not be changed.

**Review by rimi-itk — APPROVED** [review]

**Pull request merged**